src/app.ts

```diff
@@ -71,13 +71,21 @@ async function createApp(): Promise<express.Application> {
 
   app.enable("trust proxy");
 
-  app.use(loggerMiddleware);
   app.use(outboundContactUsLinksMiddleware);
 
   app.use(express.json());
   app.use(express.urlencoded({ extended: false }));
 
   app.use(cookieParser());
+  app.use(setLocalVarsMiddleware);
+  app.use(loggerMiddleware);
+
+  app.use((req, res, next) => {
+    req.log = req.log.child({
+      trace: "res.locals.trace",
```

Comment:
Is this meant to be a string instead of the variable?

Reply:
Changed to reference the variable.